internal/listener/listener.go

```diff
@@ -34,13 +34,26 @@ func NewListener(db *database.DB, runtimeConfig *config.RuntimeConfig, logs *log
 		logs:          logs,
 		runtimeConfig: runtimeConfig,
 	}
+
+	debugMux := http.NewServeMux()
+	debugMux.HandleFunc("/dump-config", l.DumpConfig)
+	debugMux.HandleFunc("/dump-incidents", l.DumpIncidents)
+	debugMux.HandleFunc("/dump-schedules", l.DumpSchedules)
```

Comment:
Although this may be a bit out of scope, could you please add a short API doc header to these three methods stating that they are used as `/debug` endpoints and that authorization needs to be checked beforehand, especially if the `checkDebugPassword` call is removed as I suggested in my other comment within `debugAuth` below.